src/escape.rs

```diff
@@ -0,0 +1,90 @@
+//! Escape characters that may have special meaning in a shell, including spaces.
+//! This is a modified version of the [`shell-escape::unix`] module of [`shell-escape`] crate.
```

Comment:
Can we submit a PR to `shell-escape` to add a version supporting `OsStr` there directly? I'm okay with us landing our own version initially, but it'd be way better if we could just use a version from them in the future.

Reply:
Great suggestion! I've initiated [a PR](https://github.com/sfackler/shell-escape/pull/9) to `shell-escape` and if it gets merged, we can avoid having a `src/escape.rs` altogether.